catkin_ws/src/kpr_interface/cfg/settings.json

```diff
@@ -0,0 +1 @@
+{"maxCurvature": "100", "maxWeight": "4400", "minWeight": "300", "armSpeed": "100"}
```

Comment:
They are strings to be able to support multiple data formats and primarily to simplify the communication.